Earthfile

```
@@ -1,12 +1,8 @@
-VERSION \
-	--global-cache \
-	--use-function-keyword \
-	--arg-scope-and-set \
-	0.7
+VERSION 0.8
 
 IMPORT gitlab.com/wunker-bunker/ci-pipelines/earthly/cargo AS cargo
```

```suggestion
IMPORT github.com/blue-build/earthly-lib/cargo AS cargo
```

I'm gonna make us a new repo for earthly functions